perception_dataset/t4_dataset/annotation_files_generator.py

```diff
@@ -297,6 +311,20 @@ def _convert_to_t4_format(
                         mask=mask[sensor_id][frame_index],
                     )
 
+                # Surface Annotation
+                if (
+                    "two_d_segmentation" in anno.keys()
+                    and anno["category_name"] in self._surface_categories
+                ):
+                    sensor_id: int = int(anno["sensor_id"])
+                    if frame_index not in frame_index_to_sample_data_token[sensor_id]:
+                        continue
+                    self._surface_ann_table.insert_into_table(
+                        category_token=category_token,
+                        mask=mask[sensor_id][frame_index],
```

Comment:
In my understanding, `mask` will be the mask that is filled with 0 in your implementation.
c.f.
```bash
                    width: int = sample_nuim["width"]
                    height: int = sample_nuim["height"]
                    object_mask: NDArray = np.array(
                        [[0 for _ in range(height)] for __ in range(width)], dtype=np.uint8
                    )
                    object_mask = cocomask.encode(np.asfortranarray(object_mask))
                    object_mask["counts"] = repr(base64.b64encode(object_mask["counts"]))[2:]
                    mask[cam_idx].update({frame_index: object_mask})
```

```suggestion
                        mask=anno["two_d_segmentation"],
```

Reply:
@Shin-kyoto Thank you for your suggestion, and sorry for the late reaction. I updated it in https://github.com/tier4/tier4_perception_dataset/pull/164/commits/9936966aaf6f35ea9e6553e798fb613d724c2afd